frontend/src/main.ts

```diff
@@ -7,28 +7,14 @@ import App from '@/App.vue';
 import router from '@/router';
 
 import 'bootstrap';
-
-// import the fontawesome core
-import { library } from '@fortawesome/fontawesome-svg-core';
-
-// import font awesome icon component
-import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
-
-// import specific icons
-import { faTrashCan } from '@fortawesome/free-regular-svg-icons';
```

Comment:
since we removed these import, let's check if those packages are still in package.json? do we still need these packages? remove cache and test it locally to make sure everything still works

Reply:
Ok, i will!!